### PR TITLE
tune the computer vision models

### PR DESCRIPTION
**Pull request by cheungdaven**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Review by AnirudhDagar — COMMENTED** [review]
NiN still seems to be [broken](http://preview.d2l.ai/d2l-en/PR-2126/chapter_convolutional-modern/nin.html). And I know that it is probably not because of your changes, it was already flaky as reported in https://github.com/d2l-ai/d2l-en/issues/1150

Same for [TF batch norm](http://preview.d2l.ai/d2l-en/PR-2126/chapter_convolutional-modern/batch-norm.html#concise-implementation)

**Comment by cheungdaven** [comment]
@AnirudhDagar Could you help me check if it looks good to you?

**Comment by AnirudhDagar** [comment]
@cheungdaven the plots look good to me, just a small merge conflict. Should we test these a couple of times to make sure that there are no inconsistencies?

**Comment by cheungdaven** [comment]
> @cheungdaven the plots look good to me, just a small merge conflict. Should we test these a couple of times to make sure that there are no inconsistencies?
Yeah, I can reevaluate it a few times.

**Comment by cheungdaven** [comment]
@d2l-bot please rebuild

**Comment by cheungdaven** [comment]
We also tested the running time of different batch size. On Resnet with Tesla T4 GPU, we have: 11m42s (batch_size=256, epochs=15), 7m47s(batch_size=256, epochs=10), 10m23s(batch_size=128, epochs=10).
Screen Shot 2022-05-20 at 3 45 16 PM
Screen Shot 2022-05-20 at 3 57 27 PM
Screen Shot 2022-05-20 at 4 07 41 PM

**Comment by astonzhang** [comment]
Thanks! Could you improve the ResNet plot? I just ran it twice and both gave similar results as follows:

Screen Shot 2022-05-21 at 12 32 06 PM

Maybe you could try testing multiple times for the same setting to mitigate randomness effect
